src/configs/mod.rs

```diff
@@ -84,6 +84,7 @@ pub struct FullConfig<'a> {
     pub scan_timeout: u64,
     pub command_timeout: u64,
     pub add_newline: bool,
+    profiles: IndexMap<String, String>,
```

Comment:
Is there a particular reason we need an IndexMap instead of a HashMap? Is this because of something we plan to do with iteration order later, or is this a requirement placed on us by Serde?

Reply:
Nope, just because I saw a similar requirement below implemented with an IndexMap, rust n00b here 😅. If you feel HashMap is better, will use it!

Reply:
fixed